src/endpoints/disclaimer.js

```diff
@@ -29,10 +30,5 @@ export default new Endpoint({
         order: page.order
       })
     }, null)
-  },
-  mapStateToOptions: (state) => ({
-    language: state.router.params.language,
-    location: state.router.params.location
-  }),
-  shouldRefetch: (options, nextOptions) => options.language !== nextOptions.language
```

Comment:
IMO same as above. Also: Currently we do not have any chance to have multiple separate endpoint-payloads from the same endpoint available. (Like if we had an /events/:id endpoint and want to load both /events/2 and /events/3, we cannot with this architecture. Should we take this into account at all?

Reply:
Thats quite difficult with our design. Also where should the 2 or 3 come from? Currently they can only come from the redux state. So I would suggest to do lazy-loading in this case. Return an array of all events. If you need a specific one it will be loaded jit (and maybe update the redux state).

But yeah it really depends which queries we have.